docs/dependencies.md

```diff
@@ -0,0 +1,22 @@
+
+- sudo; passwordless config (needed even for root users)
+- iptables - k0sctl checks if iptables exists and installs it
+- curl - Same as above
+- uname - Present in all Unix and Linux distributions
+- command - 
+- chmod - expected to always be there
+- mktemp - expected to always be there
+- install - expected to always be there
+- grep - expected to always be there
+- mv - expected to always be there
+- rm - expected to always be there
+- cat - expected to always be there
+- hostname - expected to always be there
+- find - expected to always be there
```

Comment:
Hmm, I don't remember why "findutils" is installed on alpine. It provides "xargs" but I don't see that used either. Makefile uses `find`.

Reply:
kube itself needs `find`

Reply:
kubernetes upstream issue: https://github.com/kubernetes/kubernetes/issues/95186